LovelyCrane/ViewRouter.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewRouter.swift
+//  LovelyCrane
+//
+//  Created by Toughie on 2023/07/20.
+//
+
+import Foundation
+
+class ViewRouter: ObservableObject {
+    @Published var currentPage: CurrentPage = .AuthenticationView
+}
+
+
+enum CurrentPage {
+    case AuthenticationView
+    case MainView
+    case NicknameView
```

Comment:
Swift에서 enum의 case는 small camel case가 권장됩니다 ... 🐫 ..

```suggestion
    case authenticationView
    case mainView
    case nicknameView
```